### PR TITLE
0.22: Cloud - Deployment RBAC / Workspace RBAC 

### PR DESCRIPTION
**Pull request by samblackk**
This PR updates the `cloud/stable` docs for User permissions. Assets are already updated to `web-assets`. 
Please provide feedback to improve - this may just be a first pass.

### CODE REVIEW

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Manage User Permissions on an Astronomer Workspace"
-navTitle: "Workspace User Permissions"
-description: "Manage user roles and permissions within a single Astronomer Workspace."
+navTitle: "User Permissions"
+description: "Manage user roles and permissions."
 ---
 
 Astronomer supports role based access control (RBAC), allowing you to configure varying levels of access across all Users within your Workspace.
```
Comment (kushalmalani) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:7]: Should we rephrase the last part to

"across all Users for all Deployments within your Workspace."

Or its rather more confusing?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 2]
```diff
@@ -13,14 +13,16 @@ For details on how those levels of permission are defined and how to leverage th
...
 ## Overview
 
-Astronomer supports three levels of Workspace roles:
+Astronomer supports three levels of roles for both Workspaces and Deployments:
 
    - Admin
    - Editor
    - Viewer
 
 Each of these roles maps to a combination of permissions to both Astronomer and Airflow itself.
```
Comment (kushalmalani) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:23]: Should we add a note here stating

"Users need to be part of the workspace for them to have role access to deployments"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 3]
```diff
@@ -63,6 +65,50 @@ Viewers _cannot_ push code to a deployment.
...
+### Viewer
+
+Viewers are limited to read-only mode. They can:
+
+- Can view users in a Deployment
+- Can view deployments in a Deployment
+
+Viewers _cannot_ push code to a deployment.
+
+**Note:** By default, newly invited users are `Viewers` in a Deployment
```
Comment (kushalmalani) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:110]: Wait, they are not right? they have to be assigned the viewer role also? or else they only have workspace viewer access

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 4]
```diff
@@ -63,6 +65,50 @@ Viewers _cannot_ push code to a deployment.
...
+- Perform CRUD operations on any service account in the Deployment
+
+Editors _cannot_ manage other users in the Deployment.
+
+### Viewer
+
+Viewers are limited to read-only mode. They can:
+
+- Can view users in a Deployment
+- Can view deployments in a Deployment
```
Comment (kushalmalani) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:106]: Should we rephrase this to, "Can view Deployment"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 3 above)
Comment (samblackk) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:110]: Correct, this is a bad copy paste on my part.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 1 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:7]: @kushalmalani Yea, I think we can expand on this a bit more. I just changed it to:

> "Manage user roles and permissions across any Astronomer Workspace and all Airflow Deployments within it."

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 5]
```diff
@@ -31,9 +33,9 @@ To view roles within a Workspace, navigate to the `Users` tab.
 
 If you're a Workspace Admin, you can edit permissions by clicking into a user.
 
-![Configure Access](https://assets2.astronomer.io/main/docs/astronomer-ui/configure_access.png)
+![Configure Access](https://assets2.astronomer.io/main/docs/astronomer-ui/configure_access-0.22.png)
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:36]: My name is going to be loud and clear in our doc. Yay!

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 6]
```diff
@@ -31,9 +33,9 @@ To view roles within a Workspace, navigate to the `Users` tab.
...
 
-## Astronomer Access
+## Astronomer Workspace Access
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:38]: Should we just say Workspace Access

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 7]
```diff
@@ -63,6 +65,84 @@ Viewers _cannot_ push code to a deployment.
 
 **Note:** By default, newly invited users are `Viewers` in a Workspace.
 
+## Deployment Access
+
+### View Roles
+
+To view roles within a Deployment, navigate to the `Access` tab.
+
+![Deployment Users](https://assets2.astronomer.io/main/docs/astronomer-ui/deployment_users_0.22.png)
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:74]: We could change the screenshot to include the tabs at the top with the "Access" tab selected.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 8]
```diff
@@ -63,6 +65,84 @@ Viewers _cannot_ push code to a deployment.
...
+
+### Edit Roles
+
+If you're a Deployment Admin, you can edit permissions using the dropdown.
+
+![Configure Deployment Access](https://assets2.astronomer.io/main/docs/astronomer-ui/deployment_users_edit_0.22.png)
+
+## Astronomer Deployment Access
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:82]: Title seems redundant to a previous one "Deployment Access"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 9]
```diff
@@ -63,6 +65,84 @@ Viewers _cannot_ push code to a deployment.
...
+
+- Perform CRUD (create, read, update, delete) operations on the Deployment
+- Perform CRUD Airflow operations
+- Manage users and their permissions in a Deployment
+
+### Editor
+
+Behind admins, the Editor can:
+
+- Perform CRUD operations on in the Deployment
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:96]: Typo - "on in"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 10]
```diff
@@ -63,6 +65,84 @@ Viewers _cannot_ push code to a deployment.
...
+
+## Astronomer Deployment Access
+
+### Admin
+
+Deployment Admins are the highest-tiered role. Admins can:
+
+- Perform CRUD (create, read, update, delete) operations on the Deployment
+- Perform CRUD Airflow operations
+- Manage users and their permissions in a Deployment
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:90]: Are you not mentioning service accounts in here?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 11]
```diff
@@ -63,6 +65,84 @@ Viewers _cannot_ push code to a deployment.
...
+![Configure Deployment Access](https://assets2.astronomer.io/main/docs/astronomer-ui/deployment_users_edit_0.22.png)
+
+## Astronomer Deployment Access
+
+### Admin
+
+Deployment Admins are the highest-tiered role. Admins can:
+
+- Perform CRUD (create, read, update, delete) operations on the Deployment
+- Perform CRUD Airflow operations
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:89]: Will it be clear to users, what Airflow operations means?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 4 above)
Comment (vishwas-astro) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:106]: We could say something like "Can access deployment in read-only mode"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 12]
```diff
@@ -63,6 +65,84 @@ Viewers _cannot_ push code to a deployment.
...
+Behind admins, the Editor can:
+
+- Perform CRUD operations on in the Deployment
+- Perform CRUD operations on any service account in the Deployment
+
+Editors _cannot_ manage other users in the Deployment.
+
+### Viewer
+
+Viewers are limited to read-only mode. They can:
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:103]: We should not say "They can:" and every item in the list after start with 'can'.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 1 above)
Comment (vishwas-astro) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:7]: 👍

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 13]
```diff
@@ -63,6 +65,84 @@ Viewers _cannot_ push code to a deployment.
...
+
+**Note:**
+#### Current deployment roles:
+```
+"DEPLOYMENT_VIEWER", "DEPLOYMENT_EDITOR", "DEPLOYMENT_ADMIN"
+```
+
 ## Airflow Access
 
 Astronomer RBAC not only applies to functions on Astronomer itself, but it also maps to Airflow native roles and permissions. User roles apply to all Airflow deployments within a single Workspace.
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:148]: I think this is no longer true right @kushalmalani @samblackk ?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 14]
```diff
@@ -1,41 +1,156 @@
...
-description: "Manage user roles and permissions."
+description: "Manage user roles and permissions across any Astronomer Workspace and all Airflow Deployments within it."
 ---
 
-Astronomer supports role based access control (RBAC), allowing you to configure varying levels of access across all Users within your Workspace.
+## Overview
 
-The Astronomer image comes bundled with an [astronomer-fab-security manager package](https://github.com/astronomer/astronomer-fab-securitymanager/blob/3067c0470dbb1a528d1e251fb45312d01989e6a4/README.md) that connects permissions between
-Houston (the Astronomer API) and Airflow's built-in RBAC.
+Astronomer supports a role-based access control (RBAC) and permissions framework that allows users to configure varying levels of access both at the Workspace and Deployment levels. Individual users on Astronomer can be configured to have some level of Workspace-specific permissions while holding a different set of permissions for any individual Airflow Deployment.
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:9]: "for any individual Airflow Deployment within that workspace" may be.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 5 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:115]: Hahaha

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 14 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:9]: I like that - will change @vishwas-astro !

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 15]
```diff
@@ -1,41 +1,156 @@
...
+- Run `$ astro workspace list`
+- Find it in the Workspace URL from your browser after the `/w/` (e.g. `https://app.gcp0001.us-east4.astronomer.io/w/<workspace-id>`)
+
+To set a **Role**, add a flag in the following format:
+
+- `--WORKSPACE_EDITOR`
+- `--WORKSPACE_VIEWER`
+- `--WORKSPACE_ADMIN`
+
+If you do _not_ specify a role in this command, `WORKSPACE_VIEWER` will be set by default and no deployment-level role will be assumed.
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:61]: Afaik, deployment level role will not be assumed regardless of whether role is WORKSPACE_VIEWER WORKSPACE_EDITOR or WORKSPACE_ADMIN.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 16]
```diff
@@ -1,41 +1,156 @@
...
+
+### Invite to Deployment
 
-Astronomer supports three levels of roles for both Workspaces and Deployments:
+The ability to invite users to an Airflow Deployment is limited to Deployment _Admins_, who can also grant the _Admin_ role to other users. Deployment _Editors_ and _Viewers_ cannot invite or otherwise manage users. A user who creates a Deployment is automatically granted the _Admin_ role within it.
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:65]: "invite users to an Airflow Deployment within a workspace" may be.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 15 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:61]: @vishwas-astro Yea, seems that way. Do you think that needs to be more clear here?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 16 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:65]: Good call, changed

Comment (vishwas-astro) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:65]: or you could say invite workspace users to an airflow deployment.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 17]
```diff
@@ -1,41 +1,156 @@
...
+#### via Astronomer UI
 
-## Workspace Access
+To invite a Workspace user to an Airflow Deployment via the Astronomer UI, navigate to: **Workspace** > **Deployment** > **Access**.
 
-### View Roles
+From there:
 
-To view roles within a Workspace, navigate to the `Users` tab.
+1. Type the Workspace user's name in the search bar on top
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:75]: Find the user from the list of workspace users - use the search bar on top if needed to search by name.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 18]
```diff
@@ -1,41 +1,156 @@
...
-   - Admin
-   - Editor
-   - Viewer
+> **Note:** In order for a user to be granted access to an Airflow Deployment, they must _first_ be invited to and assigned a role within the Workspace. On the other hand, a user could be a part of a Workspace but have no access or role to any Airflow Deployments within it.
 
-Each of these roles maps to a combination of permissions to both Astronomer and Airflow itself.
+#### via Astronomer UI
 
-## Workspace Access
+To invite a Workspace user to an Airflow Deployment via the Astronomer UI, navigate to: **Workspace** > **Deployment** > **Access**.
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:71]: this does not look right. You want to go to **Workspace** > **Users**. Right?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 19]
```diff
@@ -1,41 +1,156 @@
...
 
-### View Roles
+From there:
 
-To view roles within a Workspace, navigate to the `Users` tab.
+1. Type the Workspace user's name in the search bar on top
+2. Select a role from the drop-down menu on the right
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:76]: on the right of what. May be on the right of the "selected user"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 17 above)
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:75]: or click on Show All and select the user from the list of all workspace users

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 15 above)
Comment (vishwas-astro) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:61]: Reason I am calling it out is because you are mentioning it only in the context of when one does not specify a role in this command.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 20]
```diff
@@ -1,39 +1,156 @@
...
+
+1. Type the Workspace user's name in the search bar on top
+2. Select a role from the drop-down menu on the right
+3. Click the `+`
+
+[INSERT DEPLOYMENT USER INVITE GIF]
+
+#### via Astronomer CLI
+
+To invite a user to a Deployment as a Deployment _Admin_ via the Astronomer CLI, run:
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:83]: "To invite a workspace user" may be?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 21]
```diff
@@ -1,39 +1,156 @@
...
-Each of these roles maps to a combination of permissions to both Astronomer and Airflow itself.
+### Workspace
 
-### View Roles
+#### View Workspace Roles
 
-To view roles within a Workspace, navigate to the `Users` tab.
+To view roles within a Workspace via the Astronomer UI, navigate to **Workspace** > **Users**. All Workspace users have access to this view and can see the roles of all other users.
 
 ![Users](https://assets2.astronomer.io/main/docs/astronomer-ui/users_permissions.png)
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:110]: What is the CLI way of doing the same?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 22]
```diff
@@ -1,39 +1,156 @@
...
 
-### Edit Roles
+#### Edit Workspace Roles
 
-If you're a Workspace Admin, you can edit permissions by clicking into a user.
+If you're a Workspace Admin, you can edit both Workspace and Deployment-level permissions by clicking into an individual user.
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:113]: We seem to be using permissions and roles interchangeably. Not opposed to it, but just stating that i noticed it.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 23]
```diff
@@ -1,39 +1,156 @@
...
 
-![Configure Access](https://assets2.astronomer.io/main/docs/astronomer-ui/configure_access.png)
+![Configure Access](https://assets2.astronomer.io/main/docs/astronomer-ui/configure_access-0.22.png)
 
-## Astronomer Access
+### Deployment
+
+#### View Deployment Roles
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:119]: View Deployment Roles of Users as a title may be?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 24]
```diff
@@ -1,39 +1,156 @@
...
-   - Admin
-   - Editor
-   - Viewer
+## View and Edit User Roles
 
-Each of these roles maps to a combination of permissions to both Astronomer and Airflow itself.
+### Workspace
 
-### View Roles
+#### View Workspace Roles
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:105]: "View Workspace Roles of Users" as title may be

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 25]
```diff
@@ -1,39 +1,156 @@
...
 
-To view roles within a Workspace, navigate to the `Users` tab.
+To view roles within a Workspace via the Astronomer UI, navigate to **Workspace** > **Users**. All Workspace users have access to this view and can see the roles of all other users.
 
 ![Users](https://assets2.astronomer.io/main/docs/astronomer-ui/users_permissions.png)
 
-### Edit Roles
+#### Edit Workspace Roles
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:111]: Edit Workspace Roles of Users as title may be?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 4 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:207]: Clarified this!

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 6 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md]: Yea, changed these headers a bit

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 8 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:183]: +1

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 9 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md]: Fixed, good catch

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 10 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:191]: Added

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 11 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:190]: Added some context to this one, good call

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 12 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:204]: Good call, fixed

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 13 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:215]: Yea, this "Airflow Access" section is redundant/no longer accurate - ended up removing this

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 15 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:61]: Gotcha, yea good call. Changed this to:

> If you do _not_ specify a role in this command, `WORKSPACE_VIEWER` will be set by default. In all cases where a user is invited to a Workspace and deployment-level role is not specified, no deployment-level role will be assumed.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 17 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:75]: Added

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 21 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:110]: Added

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 23 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:119]: Ended up changing this to "#### View Deployment Users", which I think is ok?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 24 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:105]: Same here - changed to "View Workspace Users" instead of just "Roles"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 25 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:111]: I did "Edit Workspace User Role" and "Edit Deployment User Role"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 20 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:83]: Yea, I can be more explicit here - added!

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 19 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:76]: Yea, can also be more explicit here! Changed

**Review by paolaperaza — APPROVED** [review]
Alright! Thanks @vishwas-astro for all the comments here. I just did a few read-throughs and think that's all the content we need to add.

Want to give it a final look? One screenshot is weirdly not rendering for me - I'll make sure to fix that before merging.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 22 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md]: This is a good point - we should be consistent about this. I haven't thought about it/cleaned it up - will do so

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 26]
```diff
@@ -1,86 +1,237 @@
...
+
+![Invite Deployment User](https://assets2.astronomer.io/main/docs/astronomer-ui/invite-user-deployment.gif)
+
+#### via Astronomer CLI
+
+To invite a Workspace user to an Airflow Deployment as a Deployment _Admin_ via the Astronomer CLI, run:
 
-Astronomer supports three levels of Workspace roles:
+```
+$ astro deployment user add <email-address> --deployment-id=<deployment-id> --role=<deployment-role>
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:85]: If the example is to set the invited user's deployment role to be Deployment Admin, then make that specific in the CLI example, not just say --role=<deployment-role>

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 27]
```diff
@@ -1,86 +1,237 @@
...
+
+To invite a Workspace user to an Airflow Deployment via the Astronomer UI, navigate to: **Workspace** > **Deployment** > **Access**.
+
+From there:
+
+1. Type the Workspace user's name in the search bar on top (or click **Show All** to view all users)
+2. Select a deployment role from the drop-down menu to the right of the selected user
+3. Click the `+` symbol
+
+![Invite Deployment User](https://assets2.astronomer.io/main/docs/astronomer-ui/invite-user-deployment.gif)
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:78]: This gif is not rendering properly.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 28]
```diff
@@ -1,86 +1,237 @@
...
+
+A user who creates a Workspace is automatically granted the _Admin_ role for the Workspace and has the ability to create any number of Airflow Deployments within it.
+
+#### via Astronomer UI
+
+To invite a user to a Workspace via [the Astronomer UI](https://app.gcp0001.us-east4.astronomer.io/), navigate to **Workspace** > **Users** > **Invite User**.
+
+When a Workspace _Admin_ invites a user to a Workspace in which one or more Airflow Deployments exist, they'll have the opportunity to set that user's deployment-level roles as well, though it is not required.
+
+![Invite Workspace User](https://assets2.astronomer.io/main/docs/astronomer-ui/invite-user-workspace.gif)
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:37]: this gif is also not rendering correctly in my browser.

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 29]
```diff
@@ -1,86 +1,237 @@
...
+
+- Can access and make changes to the Workspace in the **Settings** tab
+- Can perform CRUD operations on any Service Account in the Workspace
+- Can create Airflow Deployments in the Workspace
+
+A Workspace _Editor_ cannot manage other users in the Workspace and _cannot_ add, edit or remove information in the **Billing** tab.
+
+#### Workpace Viewer
+
+A Deployment _Viewer_ is limited to read-only mode. _Viewers_:
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:204]: you mean a workspace viewer?

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md [hunk 30]
```diff
@@ -1,86 +1,237 @@
...
+
+### Workspace
+
+#### Workspace Admin
+
+Workspace _Admins_ are the highest-tiered role. Admins:
+
+- Can manage users and their permissions in a Workspace
+- Can perform CRUD (create, read, update, delete) operations on the Workspace
+- Can perform CRUD operations on any Airflow Deployment within that Workspace
```
Comment (vishwas-astro) [inline cloud/stable/05_manage-astronomer/01_workspace-permissions.md:190]: I dont think this is true any more .. What if they are a workspace admin but a deployment viewer on a deployment within it?

Comment (vishwas-astro) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:190]: Also they should be able to do CRUD operations on any service account too, right?

**Comment by vishwas-astro** [comment]
@paolaperaza i am going to approve this PR conditionally. I am assuming you will address my comments or at least find answers to my questions.

**Review by vishwas-astro — APPROVED** [review]

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 26 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:85]: Ah, no here I was trying to say if _you_ were a Deployment admin. I clarified that you had to be a Deployment admin above so I just took that out for clarify and kept the command "general"

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 30 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:190]: Yea, I should've known to fix this - thanks @vishwas-astro ! I added a note about deployment-level roles for workspace admins and added service account line-item

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 29 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:204]: Whoops, typo - thanks - fixed!

**Review by paolaperaza — APPROVED** [review]

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 27 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:78]: Hm, it is on mine - might have to merge and double check all of these

File: cloud/stable/05_manage-astronomer/01_workspace-permissions.md (hunk 28 above)
Comment (paolaperaza) [reply cloud/stable/05_manage-astronomer/01_workspace-permissions.md:37]: same here - it's working for me now!

**Comment by paolaperaza** [comment]
Alright! Going ahead and merging this 😊
